File: python/run-local.py
from run_morph_kgc import write_file

# result = write_file("transformation/mappings/s1_contracting_authority.rml.ttl", 'output-notice.nt')

result = write_file("../simple/characters.rml.ttl",'output.nt')

print(f"Graph created with {result} triples.")

File: python/run-local.js
import { createMapper } from "./mapper.js";

const triplify = await createMapper();

// const result = await triplify({
//   mappingFile: '../simple/characters.rml.ttl', outputFile: 'output-local.nt',
// })

const result = await triplify({
  mappingFile: "../simple/characters.rml.ttl",
  outputFile: "output-local",
});
console.log(result);
